### PR TITLE
Some demo project license fixes

### PR DESCRIPTION
**Pull request by zonkmachine**
Some license fixes.

Tracked down some missing licenses and split out licenses from demos/LICENSES.TXT for projects that were moved to new directories, /Skiessi and /CapDan.

### CODE REVIEW

**Comment by zonkmachine** [comment]
I looked into the other project directories. CapDans and Skiessis projects were given their own directories so I've moved the corresponding licenses to separate files. After the above commits there are still someProjects missing license info. Detective mode on...

/data/projecs/demos/
AngryLlama-NewFangled.mmpz
Ashore.mmpz
DnB.mmpz
Jousboxx-BuzzerBeater.mmpz
Popsip-Electric Dancer.mmpz
Shovon-ProgressiveHousePluckDemo.mmpz
StrictProduction-DearJonDoe.mmp
Thaledric-Armageddon.mmpz
TobyDox-Psycho.mmpz

/data/projecs/demos/Skiessi/
Skiessi-RandomProjectNumber14253.mmpz
Skiessi-TurningPoint.mmpz

/data/projecs/shorties/
sv-DnB-Startup.mmpz
sv-Trance-Startup.mmpz

**Comment by musikBear** [comment]
Good! I had a somewhat lengthy 'chat' on the subject (@zonkmachine You gave the right license 👍 ) But then i realized something. 
Cant write it here, so i will try 'discourt' or whatever....

**Comment by zonkmachine** [comment]
> Shovon-ProgressiveHousePluckDemo.mmpz

According to the commit message it's public domain. 03931e8b17c1eb84c6b74b86f1ad45748498adc1
We really need better procedures for this. As an example, here's how 0AD deals with contributors/Licenses. One license and everybody has to agree to it in one public thread. https://wildfiregames.com/forum/index.php?/topic/16802-legal-waiver-please-read-this-before-contributing/&page=3

**Comment by zonkmachine** [comment]
I think I'll just squash/merge what I've got here. Chasing down the remnant projects will take too much time and they mostly seem to have been taken down from the sharing platform where most of the projects came from in the first place. No objection means merging in a day.

File: data/projects/demos/LICENSES.TXT [hunk 1]
```diff
@@ -34,6 +18,10 @@
   - Artistic 2.0
   - https://lmms.io/lsp/index.php?action=show&file=581
 
+* Jousboxx-BuzzerBeater.mmpz
+  - Creative Commons (by-nc-sa)
+  - https://lmms.io/lsp/?action=show&file=10488
+
```
[the remark this answers is not in the document]
Comment (zonkmachine) [reply data/projects/demos/LICENSES.TXT]: Ah! And here he specifies the license to `CC BY-SA 4.0` https://github.com/LMMS/lmms/pull/3482#issuecomment-293609866

[the remark this answers is not in the document]
Comment (zonkmachine) [reply data/projects/demos/LICENSES.TXT]: BuzzerBeater is already in the stable-1.2 license file but it's tucked at the end and I missed it. I'll just move it to it's correct place in the alphabet.

[the remark this answers is not in the document]
Comment (zonkmachine) [reply data/projects/demos/LICENSES.TXT]: Fixed!

**Pull request merged**